README.md

```diff
@@ -1,4 +1,9 @@
-# identity-play-auth
+# identity-auth
+
+An sbt multi build project containing projects used for user authentication.
+
+## identity-play-auth
+
 A small library for Guardian Identity authentication with the Play framework
 
 * Use 2.x series for Play 2.6
```

Comment:
Here's some stuff I might want to know in a README:
1) If I make a change just to identity-auth or identity-play-auth, how do I release changes? Will it bump versions for both? Which projects need a version bump when changes are released? 
2) How do I make use of identity-auth/identity-play-auth?

Reply:
Hey @lmath - useful points, thanks 👍; documentation will follow once it is clear that (1) both these libraries will make it to the final PR; and (2) what functionality will be in these libraries vs the applications that use them.